recipes/twitch-native-ipc/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class TwitchNativeIpcConan(ConanFile):
+    name = "twitch-native-ipc"
+    license = "MIT"
+    homepage = "https://github.com/twitchtv/twitch-native-ipc"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Twitch natve ipc library"
+    topics = ("<twitch>", "<ipc>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "libuv:shared": False}
```

Comment:
```suggestion
    default_options = {"shared": False, "fPIC": True}
```
Why is `libuv:shared` defaulted to False?
People can still override to `True`, and its default is `False` anyways.
So this option is redundant.

Also, as a general rule on cci, we don't force options on dependencies in `configure`, but check the options and complain (by raising a `ConanInvalidConfiguration` exception) when some option is not to our pleasing.